presto-jmx/src/test/java/com/facebook/presto/connector/jmx/TestJmxMetadata.java

```diff
@@ -76,4 +76,22 @@ public void testGetTimeTableHandle()
         assertTrue(columns.contains(new JmxColumnHandle("Name", createUnboundedVarcharType())));
         assertTrue(columns.contains(new JmxColumnHandle("StartTime", BIGINT)));
     }
+
+    @Test
+    public void testGetCumulativeTableHandle()
```

Comment:
Add test with pattern which actually resolves to more than one Object and where objects have repeating attributes.

Reply:
`java.lang:*` resolves to more than one object. I will add assertion for that